### PR TITLE
HS-1392: Fixed the error and cucumber test: short term solution

### PR DESCRIPTION
**Pull request by yangli2014**
## Jira link(s)
- https://issues.opennms.org/browse/HS-xx

## Checklist
* [x] Follows Horizon Stream's [development guidelines.](https://github.com/OpenNMS/horizon-stream/wiki/Development-Guidelines)
* [x] Appropriate reviewer(s) have been selected.
* [x] Jira issue(s) have been updated to "In Review".
* [x] Includes [appropriate tests.](https://github.com/OpenNMS/horizon-stream/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.
* [ ] Notify devops of changes to the Charts

### CODE REVIEW

File: alert/src/main/resources/db/changelog/hs-0.1.0/data/alert_definition_test_data.xml [hunk 1]
```diff
@@ -13,270 +13,270 @@ FIXME: Figure out how to set context in test container
         <column name="uei" value="uei.opennms.org/vendor/cisco/traps/SNMP_Link_Down"/>
         <column name="reduction_key" value="%s:%s:%d"/>
         <column name="type" value="PROBLEM_WITH_CLEAR"/>
-        <column name="trigger_event_id" value="1"/>
+        <column name="trigger_event_id" value="1000"/>
```
Comment (BenD10) [inline alert/src/main/resources/db/changelog/hs-0.1.0/data/alert_definition_test_data.xml:16]: I think there have been issues before from updating liquibase commands in place, so might need to create a new migration for this update.

Comment (yangli2014) [reply alert/src/main/resources/db/changelog/hs-0.1.0/data/alert_definition_test_data.xml:16]: As I said this is for a temp solution. I will fix and change it totally with cucumber style to populate the data in Hs-1383

Comment (cgorantla) [reply alert/src/main/resources/db/changelog/hs-0.1.0/data/alert_definition_test_data.xml:16]: @yangli2014 says this test data shouldn't cause any issues for the liquibase update.

**Review by cgorantla — APPROVED** [review]

File: alert/src/main/resources/db/changelog/hs-0.1.0/data/alert_definition_test_data.xml (hunk 1 above)
Comment (yangli2014) [reply alert/src/main/resources/db/changelog/hs-0.1.0/data/alert_definition_test_data.xml:16]: Now those data are only used for cucumber tests, I will fix it to enforce the DB constraints

**Pull request merged**